deposit-app/src/test/java/edu/unc/lib/boxc/deposit/normalize/DirectoryToBagJobTest.java

```diff
@@ -112,6 +113,7 @@ public void testConversion() throws Exception {
         Bag workBag = model.getBag(work.getURI());
         Resource file = getChildByLabel(workBag, "lorem.txt");
         assertTrue(file.hasProperty(RDF.type, Cdr.FileObject), "Type was not set");
+        assertFalse(work.hasProperty(Cdr.primaryObject, file), "Primary object was not set");
```

Comment:
i think the message is wrong? if it fails then the primary object was set, right?